Return exit value in test

fangohr opened the pull request:
- this is now possible due to progress with https://github.com/joommf/joommf/issues/75
- I hope the build will now fail (and alert us to missing dependencies that we didn't catch before)

fangohr committed:
Return exit value in test

- this is now possible due to progress with https://github.com/joommf/joommf/issues/75

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

fangohr committed:
Adding missing dependencies

- These are used by our way of using pandas functions, and not part of the pandas dependencies.
- They were missing before because our tests didn't return an exit code, and thus we didn't notice the failure.
